src/Recoil.js

```diff
@@ -29,6 +29,10 @@ export type {
   RecoilValueReadOnly,
 } from './core/Recoil_RecoilValue';
 
+const {unstable_batchedUpdates} = require('./util/ReactBatchedUpdates');
+const {setBatch} = require('./util/Recoil_batch');
+setBatch(unstable_batchedUpdates);
```

Comment:
I like that this is exported; future-proofs the library a bit. We should probably add a test for this once they are passing.

Reply:
Agreed